SCF-Session-8-Assignment/src/question/Bird.java

```diff
@@ -0,0 +1,18 @@
+package question;
+
+/**
+ * This class is for category Birds
+ * @author Jugal
+ * Dated 07/23/2019
+ */
+public abstract class Bird extends Animal{
+
+	/**
+	 * Constructor to assign specialty and category
+	 */
+	public Bird()

```

Comment:
Declare some different property of the Bird class, and initialize all the property of Bird and Animal class using parameterized constructor and super().

Reply:
Done